addons/website_payment/static/src/js/website_payment_donation.js

```diff
@@ -35,4 +38,42 @@ publicWidget.registry.WebsitePaymentDonation = publicWidget.Widget.extend({
             donationCommentEl.value = "";
         }
     },
+    /**
+     * @private
+     */
+    _onChangeAmountInput() {
+        const inputEl = document.querySelector("#other_amount_value");
+        const warningMessageEl = document.querySelector("#warningMessageId");
+        const warningMinMessageEl = document.querySelector("#warningMinMessageId");
+        const warningMaxMessageEl = document.querySelector("#warningMaxMessageId");
+        const value = parseFloat(inputEl.value);
+        warningMessageEl.classList.toggle("d-none", Boolean(value));
```

Comment:
Hello @paru-odoo just a quick one do we really need Boolean IMO If value is truthy (anything that isn't false, null, 0, undefined, NaN, or an empty string), the class "d-none" will be added to the element.

Reply:
@shsa-odoo  I previously went through this process, but at that time, I didn't include some conditions, which led to the addition of a boolean. However, it is no longer required now. Thanks for pointing that out.